maven-plugins/hk2-inhabitant-generator/pom.xml

```diff
@@ -93,6 +93,7 @@
                          <configuration>
                              <excludes>
                                  <exclude>**/negative/*</exclude>
+                                 <exclude>gendir-tests.jar</exclude>
```

Comment:
What exactly it fixes? Does the build really generate this jar file to target/test-classes directory?
EDIT: yes, and also gendirCopy.jar. But I'm not sure if this is the right fix ... better would be to use different directory ... see next comment.

Reply:
Line 99
<outputDirectory>${project.build.testOutputDirectory}</outputDirectory> (resolves to target/test-classes)
should be 
<outputDirectory>${project.build.directory}</outputDirectory> (resolves to target)
(or should be removed to be default)

Reply:
Ok, I run it, it seems your fix is the simplest possible. Tests depend on the file ... but it should not be included in distributed test-jar.